src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -913,7 +915,7 @@ def can_fuse(self) -> bool:
     def make_output_quant_parameters(
         self,
         q_values: Union[numpy.ndarray, Any],
-        scale: numpy.float64,
+        scale: npt.NDArray[numpy.float64],
```

Comment:
is `npt` a common alias ? why not just put `numpy.ndarray` like usual ?

Reply:
I was `npt` in some document somewhere and that help keep the code succint.

There is no way as far as I'm aware to specify the dtype of the np array using only `np.ndarray` and thought it would be better to specify it here.

Reply:
got it thanks !